lstchain/data/lstchain_standard_config.json

```diff
@@ -37,15 +37,17 @@
     "boundary_thresh":3,
     "keep_isolated_pixels":false,
     "min_number_picture_neighbors":1,
-    "use_only_main_island":true
+    "use_only_main_island":true,
```

Comment:
According to the discussion in #628, we could change the analysis to use not only the main island by default, I think this should be the preferred option

Reply:
I agree, we'll come to that. Did not want to change more than needed for now.